docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md

```diff
@@ -1 +1,182 @@
-Ubuntu specific instructions
\ No newline at end of file
+# Installing LORIS In-Depth
+
+This guide assumes an installation on an Ubuntu server. 
+
+If you are using CentOS, please visit the CentOS installation guide for 
+instructions on installation.
+
+We also have instructions for how to set up LORIS for macOS. However, only
```

Comment:
add macos link

Reply:
https://github.com/aces/Loris/pull/5645 is changing the name and location of this file. We can amend this document to link to its new location later on.